std/experimental/allocator/building_blocks/free_list.d

```diff
@@ -1045,6 +1047,28 @@ struct SharedFreeList(ParentAllocator,
         resetNodes();
         return result;
     }
+
+    /**
+    Nonstandard function that minimizes the memory usage of the freelist by
+    freeing each element in turn. Defined only if $(D ParentAllocator) defines
+    $(D deallocate).
+    */
+    static if (hasMember!(ParentAllocator, "deallocate") && !unchecked)
```

Comment:
OK, I understand. Could you please follow up with a PR to eliminate the constraint from minimize() in both freelists? It has no place there, and minimize is so useful I'm considering adding it as an official optional primitive. Feel free to change the name `unchecked` as well, though I think it's fine (the unchecked situation is particular, the majority are checked). Thanks!